sys/net/gnrc/netif/init_devs/auto_init_sx127x.c

```diff
@@ -61,16 +58,19 @@ void auto_init_sx127x(void)
 #endif
 
         sx127x_setup(&sx127x_devs[i], &sx127x_params[i]);
-#ifdef MODULE_GNRC_NETIF_LORAWAN
-        /* Currently only one lora device is supported */
-        assert(SX127X_NUMOF == 1);
+        if (IS_USED(MODULE_GNRC_LORAWAN)) {
```

Comment:
Is it ok to use `MODULE_GNRC_LORAWAN`?

Reply:
(Instead of `MODULE_GNRC_NETIF_LORAWAN`)

Reply:
Ooops, I think I did not pay attention here.

Reply:
Fixed!